packages/cli/src/commands/client-task-queue/run-batch.ts

```diff
@@ -0,0 +1,71 @@
+import { ClientTaskQueueService, State } from '@meeco/sdk';
+import { flags as _flags } from '@oclif/command';
+import { AuthConfig } from '../../configs/auth-config';
+import { authFlags } from '../../flags/auth-flags';
+import MeecoCommand from '../../util/meeco-command';
+
+export default class ClientTaskQueueList extends MeecoCommand {
+  static description = 'Load and run a batch of ClientTasks from the queue';
+  static examples = [`meeco client-task-queue:run-batch -a path/to/auth.yaml 10`];
+
+  static flags = {
+    ...MeecoCommand.flags,
+    ...authFlags,
+    supressChangingState: _flags.string({
+      required: false,
+      default: 'true',
+      description: 'suppress transitioning tasks in the response to in_progress: true, false',
+    }),
+    state: _flags.string({
+      char: 's',
+      required: false,
+      default: 'Todo',
+      description: 'Client Task Queue avalible states: ' + Object.keys(State),
+    }),
+  };
+
+  static args = [
+    {
+      name: 'numberOfTasks',
+      description: 'number of tasks to fetch and execute',
+      required: true,
```

Comment:
Should the `default` of `5` be here? Does `args` support a `default` ?

Reply:
actually the server has a default page size so I'll just make this optional instead of putting a default